src/verilator/VerilatorDinoCPU.hh

```diff
@@ -0,0 +1,31 @@
+
+#ifndef __VERILATOR_VERILATOR_OBJECT_HH__
+#define __VERILATOR_VERILATOR_OBJECT_HH__
+
+#define VM_TRACE 0
+#define VL_THREADED 0
+
+#include "VTop.h"
+#include "VerilatorMemBlackBox.hh"
+#include "params/VerilatorDinoCPU.hh"
+#include "sim/sim_object.hh"
+
+class VerilatorDinoCPU
+{
+    private:
+        void updateCycle();
+
+        VTop top;
+        VerilatorMemBlackBox verilatorMem;
```

Comment:
Should probably be a pointer.

Also, to get this pointer into this object, you can add a parameter to the SimObject description file that requires VerilatorMemBlackBox object.